tst/EnergyPlus/unit/AirflowNetworkHVAC.unit.cc

```diff
@@ -5957,6 +5959,8 @@ TEST_F(EnergyPlusFixture, AirflowNetwork_MultiAirLoopTest)
     state->dataEnvrn->OutBaroPress = 99063.0;
     state->dataEnvrn->WindSpeed = 4.9;
     state->dataEnvrn->WindDir = 270.0;
+    state->dataEnvrn->StdRhoAir = 1.2;
```

Comment:
It would probably be a good idea to collect things that are extremely common in the unit tests and pull them into the main fixture setUp routine.